node/src/chain_spec/halongbay.rs

```diff
@@ -22,6 +23,8 @@ fn halongbay_staging_testnet_config_genesis(wasm_binary: &[u8]) -> halongbay::Ge
 	let endowed_accounts = vec![
 		// 5HNFRkCYoriHQwuJbt5YgSwegRTxmSQRe51UKEEBWnUZuHf5
 		hex!["ea8e9d3cfedc8afec25785703681d424e6aba10b728927b89d87a3776b47ee32"].into(),
+		// init Crowdloan fund for testing purpose
+		halongbay::CrowdloanPalletId::get().into_account(),
```

Comment:
don't add it to the endowed account. will increase total supply.
Add to genesis of pallet_crowdloan and in genesis will deposit balance into this account

Reply:
Removed it at
https://github.com/polkafoundry/newpolka/pull/55/commits/c67c885a9d1f03eaa8f958b24750eee4a6531327#diff-6f4a120fd784e4a079ef250a3a32186861aad17023f40bc8e6a654d429d63b43L27